### PR TITLE
[1LP][RFR] added check for ansible logs

### PR DESCRIPTION
**Pull request by lina-nikiforova**
{{ pytest: -v cfme/tests/configure/test_log_depot_operation.py::test_collect_log_depot }}

1. automated test_log_collect_ansible_logs (check that downloaded logs contain ansible logs)
2. removed old blocker
3. added new blocker for new BZ

### CODE REVIEW

File: cfme/tests/configure/test_log_depot_operation.py [hunk 1]
```diff
@@ -183,12 +196,88 @@ def check_ftp(ftp, server_name, server_zone_id):
...
+def catalog(appliance, ansible_catalog_item):
+    catalog_ = appliance.collections.catalogs.create(fauxfactory.gen_alphanumeric(),
+                                                     description='my catalog',
+                                                     items=[ansible_catalog_item.name])
+    ansible_catalog_item.catalog = catalog_
+    yield catalog_
+
+    if catalog_.exists:
+        catalog_.delete()
+        ansible_catalog_item.catalog = None
```
Comment (digitronik) [inline cfme/tests/configure/test_log_depot_operation.py]: :+1:

File: cfme/tests/configure/test_log_depot_operation.py [hunk 2]
```diff
@@ -183,12 +196,88 @@ def check_ftp(ftp, server_name, server_zone_id):
...
+
+
+@pytest.fixture
+def service_request(appliance, ansible_catalog_item):
+    request_descr = "Provisioning Service [{0}] from [{0}]".format(ansible_catalog_item.name)
```
Comment (digitronik) [inline cfme/tests/configure/test_log_depot_operation.py]: Optional:
In such cases, I will prefer to use `key` than `index` like
`request_descr = "Provisioning Service [{name}] from [{name}]".format(name=ansible_catalog_item.name)`

File: cfme/tests/configure/test_log_depot_operation.py [hunk 3]
```diff
@@ -13,12 +13,13 @@
 
 from cfme import test_requirements
 from cfme.utils import conf, testgen
+from cfme.utils.ssh import SSHClient
```
[the remark this answers is not in the document]
Comment (lina-nikiforova) [reply cfme/tests/configure/test_log_depot_operation.py]: done

File: cfme/tests/configure/test_log_depot_operation.py [hunk 4]
```diff
@@ -183,12 +196,88 @@ def check_ftp(ftp, server_name, server_zone_id):
...
+
+@pytest.fixture(scope="module")
+def ansible_repository(appliance, wait_for_ansible):
+    repositories = appliance.collections.ansible_repositories
+    repository = repositories.create(
+        name=fauxfactory.gen_alpha(),
+        url="https://github.com/quarckster/ansible_playbooks",
+        description=fauxfactory.gen_alpha())
+    view = navigate_to(repository, "Details")
+    if appliance.version < "5.9":
```
[the remark this answers is not in the document]
Comment (lina-nikiforova) [reply cfme/tests/configure/test_log_depot_operation.py]: there's no refresh button in 5.8 but it is there in 5.9 and 5.10

File: cfme/tests/configure/test_log_depot_operation.py [hunk 5]
```diff
@@ -169,6 +170,18 @@ def check_ftp(ftp, server_name, server_zone_id):
         try:
             date_from = datetime.strptime(date_from, "%Y%m%d%H%M%S")
             date_to = datetime.strptime(date_to, "%Y%m%d%H%M%S")
+            # if the file is correct, check ansible logs (~/ROOT/var/log/tower/setup-*) are there
+            if ftp.login != 'anonymous':  # can't login as anon using SSH
+                with SSHClient(hostname=ftp.host,
+                               username=ftp.login,
+                               password=ftp.password) as log_ssh:
+                    result = log_ssh.run_command(
+                        "unzip -l ~{} | grep ROOT/var/log/tower/setup".format(file.path),
```
[the remark this answers is not in the document]
Comment (lina-nikiforova) [reply cfme/tests/configure/test_log_depot_operation.py]: done

File: cfme/tests/configure/test_log_depot_operation.py (hunk 2 above)
Comment (lina-nikiforova) [reply cfme/tests/configure/test_log_depot_operation.py]: yep, looks better, I changed

File: cfme/tests/configure/test_log_depot_operation.py (hunk 4 above)
[the remark this answers is not in the document]
Comment (digitronik) [reply cfme/tests/configure/test_log_depot_operation.py]: @lina-nikiforova then please remove it. we are not supporting 5.8 against the `master` branch. If you like you can add a path in `5.8` branch

**Pull request merged**